.github/workflows/build.yml

```diff
@@ -391,9 +391,10 @@ jobs:
           path: coverage
 
       - name: Codecov
-        uses: codecov/codecov-action@ab904c41d6ece82784817410c45d8b8c02684457 # v3.1.6
+        uses: codecov/codecov-action@e0b68c6749509c5f83f984dd99a76a1c1a231044 # v4.0.1
```

Comment:
@viceice did you see this breaking change in the changelog?

> The Codecov Action runs as a node20 action due to node16 deprecation. See [this post from GitHub](https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/) on how to migrate.

I think our build workflow still uses node 18?

Reply:
the actions are running on a buildin node version, so this is safe.

Reply:
I'll let you resolve this comment then. 😄